app/src/main/res/layout/add_to_cart_dialouge_layout.xml

```diff
@@ -0,0 +1,111 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:background="@drawable/bottom_nav_background"
+    android:padding="@dimen/_8sdp"
+    tools:context=".ui.product_details_screen.dialog.AddToCartDialog">
+
+    <TextView
+        android:id="@+id/txtProductTitle"
+        style="@style/fontBoldStyle"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:textSize="@dimen/fontHeading2"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
+        tools:text="@string/select_product_quantity" />
+
+    <androidx.constraintlayout.widget.ConstraintLayout
+        android:id="@+id/constrain_product_quantity"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="@dimen/_9sdp"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/txtProductTitle">
+
+        <ImageButton
```

Comment:
CardView that wraps the image would be much better option

Reply:
It was ugly when I did It.

Reply:
as you like but card view contains a lot of attributes that will make your button looks much better,
I would suggest adding a ripple effect at least

Reply:
Ok, I will change it.